### PR TITLE
Add Restore health and Kamui to Genji Days Of The Blade BCES00002 01.00

### PR DESCRIPTION
**Pull request by khangly**
This adds two cheat codes to Genji Days Of The Blade [BCES-00002]:
* Restore health
* Restore Kamui

### CODE REVIEW

File: docs/codes/Genji Days Of The Blade BCES00002 01.00.ncl [hunk 1]
```diff
@@ -1,6 +1,14 @@
 Also known as
 Genji Kamiu Souran
 
+#
+Restore health
+0
+F2 10F8BAAC 10F8BAAE
```
Comment (bucanero) [inline docs/codes/Genji Days Of The Blade BCES00002 01.00.ncl:7]: that `F2` code seems unsupported by Artemis... maybe it's for a different tool like netcheat ?

Comment (khangly) [reply docs/codes/Genji Days Of The Blade BCES00002 01.00.ncl:7]: It's from the documentation of the [netcheat file provided by you](https://github.com/bucanero/ArtemisPS3/blame/master/docs/INSTALLATION.md#L38) and also appears in [some other cheat codes](https://github.com/bucanero/ArtemisPS3/blob/master/docs/codes/Shin%20Sangoku%20Muou%20MultiRaid%20Special%20BLAS50163%2001.01.ncl#L28).

Comment (bucanero) [reply docs/codes/Genji Days Of The Blade BCES00002 01.00.ncl:7]: Yes, my bad, I haven't been reviewing Artemis in a long time.

File: docs/codes/Genji Days Of The Blade BCES00002 01.00.ncl [hunk 2]
```diff
@@ -1,6 +1,14 @@
...
+#
+Restore Kamui
+0
```
Comment (bucanero) [inline docs/codes/Genji Days Of The Blade BCES00002 01.00.ncl:10]:
```suggestion
0
khangly
```

File: docs/codes/Genji Days Of The Blade BCES00002 01.00.ncl (hunk 1 above)
Comment (bucanero) [reply docs/codes/Genji Days Of The Blade BCES00002 01.00.ncl:8]: These additional code types (like `F`) should be added to the Wiki:
https://github.com/bucanero/ArtemisPS3/wiki/Artemis-NetCheat-code-types

They are currently missing from the documentation.

**Review by bucanero — APPROVED** [review]

**Pull request merged**